src/App.tsx

```diff
@@ -1,204 +1,168 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useRef, useState } from 'react';
+// import { UserWarning } from './UserWarning';
```

Comment:
Remove all comments
```suggestion
```

Reply:
sorry but this comments will need me in future tasks. I don't want to remove it.